inst/emails/contact_initial_aon

```diff
@@ -2,20 +2,25 @@ Dear %s,
 
-At this time, please note that NSF requires all metadata created during your project be submitted to a national data center or another long-lived, publicly accessible archive within two years of collection or before the end of your award, whichever comes first. Additionally, data and derived data products are required to be submitted within 6 months of collection. More detailed information about what will be expected of you can be found at https://arcticdata.io/submit/#who-must-submit.
-
-With these requirements in mind, we strongly recommend that you plan to keep detailed and well-documented data and metadata records throughout your project. Typically, NSF researchers already do a great job of collecting data and metadata, but you will want to ensure your data and metadata are collected in a manner that can be easily accessed and understood by other researchers after it is published. We recommend that you visit our data submission guidelines page (https://arcticdata.io/submit/) to get a better understanding of what is expected for submissions to the Arctic Data Center.
-
-Please note that for AON program projects, NSF requires that:
-    - Real-time data be made publicly available immediately. If there is any question about what constitutes real-time data, please contact the appropriate NSF program officer.
+Please note that for AON program projects, NSF has the additional requirements:
+    - Real-time data be made publicly available immediately. 
     - All data be submitted to a national data center or another long-lived, publicly accessible archive within 6 months of collection, and be fully quality controlled.
     - All data sets and derived data products be accompanied by a metadata profile and full documentation that allows the data to be properly interpreted and used by other researchers.
 
-If you have any questions about the Arctic Data Center or NSF requirements, please reply to this email and we will respond as soon as possible. We will also contact you again near the end of your award with reminders about the data submission requirements.
+More detailed information about what will be expected of you can be found at:
+https://arcticdata.io/submit/#who-must-submit.
+
+With these requirements in mind, we strongly recommend that you plan to keep detailed and well-documented data and metadata records throughout your project. Some guidelines are provided here: https://arcticdata.io/submit/#organizing-your-data for best practices. For those who are interested, we also have annual training opportunities posted here: https://arcticdata.io/training/.
+
+If your research will involve large submissions of data (a total size of 500MB or more), please get in touch with us so we can create a plan together for those datasets.
```

Comment:
Same issue as above.

Reply:
Also, note that AON data is often semi-automated from instruments, and that it may be worthwhile for them to use our API to deposit data files from a script automatically as it is generated and/or processed.  We can work with them on how to use the API to automate their processes. I think we should mention this in here somewhere.